holoviews/operation/datashader.py

```diff
@@ -1334,12 +1334,12 @@ class rasterize(AggregationOperation):
     """
 
     aggregator = param.ClassSelector(class_=(ds.reductions.Reduction, basestring),
-                                     default=None)
+                                     default='default')
 
     interpolation = param.ObjectSelector(
-        default='bilinear', objects=['linear', 'nearest', 'bilinear', None, False], doc="""
+        default='default', objects=['default', 'linear', 'nearest', 'bilinear', None, False], doc="""
         The interpolation method to apply during rasterization.
-        Defaults to linear interpolation and None and False are aliases
+        Default depends on element type, None and False are aliases
```

Comment:
```suggestion
        Default depends on element type; None and False are aliases
```

Is that really true? It seems to require the string 'default' now, not None or False.